pyvo/make_vid.py

```diff
@@ -168,4 +182,18 @@ def make_pyvo(
 
     print(result.graph)
 
+    outname = [date.strftime('%Y-%m-%d'), event, speaker, title,
+               'preview' if preview else None]
+    outname = filter(lambda x: x, outname)
+    outname = [slugify(x) for x in outname]
```

Comment:
Nitpick: `[slugify(x) for x in outname if x]` should be shorter and faster than pre-`filter`-ing